packages/server/src/api/routes/application.ts

```diff
@@ -21,6 +21,7 @@ router
   )
   .get("/api/applications/:appId/definition", controller.fetchAppDefinition)
   .get("/api/applications", controller.fetch)
+  .get("/api/applications/client", controller.fetchClientApps)
```

Comment:
I'm not sure about this endpoint name - its not clear what this does, would it be worth creating a new resource `/api/workspace/apps` to start expanding upon? I think at some stage in the future we're going to need to update the APIs to use the workspace word as we need the "apps" word back.

Reply:
After discussing it, renamed to `/api/client/applications`